src/modules/cpu/kernel/down_mixing.hpp

```diff
@@ -0,0 +1,49 @@
+#include "rppdefs.h"
+
+RppStatus down_mixing_host_tensor(Rpp32f *srcPtr,
+                                  RpptDescPtr srcDescPtr,
+                                  Rpp32f *dstPtr,
+                                  Rpp64s *samplesPerChannelTensor,
+                                  Rpp32s *channelsTensor,
+                                  bool  normalizeWeights)
+{
+    for(int batchCount = 0; batchCount < srcDescPtr->n; batchCount++)
```

Comment:
Why are we not parallelizing across multiple threeads?

Reply:
Done